rfcs/005-group-stats.md

```diff
@@ -0,0 +1,79 @@
+# Summary
+A page at the course-instance level for group statistics. 
+
+# Basic Example Workflow
+An instructor with appropriate access uploads a reference (i.e. desired) groups assignment for their class. PrairieLearn will display group-level statistics across assessments in a given course instance. Instructor can then decide to adjust filters and parameters to get desired statistics and be able to recompute and download. 
+
+*Group Participation* 
```

Comment:
It's intended on being attendance-based, so the definition here would be "on a given group assignment, the student was in a group with (a subset of) their assigned group", according to the supplied group assignments file.

However, ability to add/adjust parameters here would also be nice, such as if a student's group didn't show up to class for an in-class assignment, but the student worked on the assignment during class, having the ability to include "worked alone/with another group during class period."

Reply:
Gotcha! "Participation" to me implies some amount of action. Maybe calling this "Group Membership" would be better?

A thought that came to mind as thinking about this specific feature: I might consider calling this feature as a whole "Group Insights" instead of "Group Statistics"?

Reply:
I completely agree on both thoughts! I think those both do a much better job of capturing the intent of that table and the intent of the feature as a whole!